orttraining/orttraining/python/training/artifacts.py

```diff
@@ -207,11 +209,17 @@ def _export_to_ort_format(model_path, output_dir, extra_options):
 
     logging.info("Optimizer enum provided: %s", optimizer.name)
 
+    opset_version = None
+    for domain in model.opset_import:
+        if domain.domain == "":
```

Comment:
Should this be domain.domain == "" or domain.domain=="ai.onnx" ?

Reply:
I'll add it but I never saw ai.onnx ever used.